src/main/resources/db/migration/V3_31__add_index_on_booking_reference_column_for_visit_notification_event_table.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX idx_visit_notification_event_booking_reference ON visit_notification_event(booking_reference);
```

Comment:
Missing trailing newline

Reply:
I've accidentally committed this! I'll revert it, this is for another PR. Thanks Paul!